src/index.ts

```diff
@@ -17,15 +17,20 @@
 
 import * as tfc from '@tensorflow/tfjs-core';
 
-import {nodeFileSystemRouter} from './io/file_system';
-import {NodeJSKernelBackend} from './nodejs_kernel_backend';
+import { nodeFileSystemRouter } from './io/file_system';
+import { NodeJSKernelBackend } from './nodejs_kernel_backend';
 
 // tslint:disable-next-line:no-require-imports
 import bindings = require('bindings');
-import {TFJSBinding} from './tfjs_binding';
+import { TFJSBinding } from './tfjs_binding';
 
 tfc.ENV.registerBackend('tensorflow', () => {
-  return new NodeJSKernelBackend(bindings('tfjs_binding.node') as TFJSBinding);
+  const backend =
+    new NodeJSKernelBackend(bindings('tfjs_binding.node') as TFJSBinding);
+  if (backend.binding.TF_Version === undefined) {
```

Comment:
Don't worry about this - if the binding fails to load here, an exception is handled in tfjs-core. Let's drop this change.

Reply:
Ack. Thanks for checking!